cockroachcloud/authorization.md

```diff
@@ -1,44 +1,165 @@
+
+### What roles are assigned to the user that creates a CockroachDB Cloud organization and thus becomes the first and only user in that organization?
+
+Org Member, Org Admin, Cluster Admin and Billing Coordinator [to be added soon] are assigned to the first and only user in a CockroachDB Cloud organization. This is done to allow the user perform all possible actions to create & manage clusters, configure billing & view usage etc. 
+
+Once the initial user has added more users to the CockroachDB Cloud organization, it is possible to assign Cluster Admin and Billing Coordinator roles to one or more of those users and optionally remove those roles from the initial user. Latter operation is allowed to satisfy enterprise security standards as some companies like to differentiate between product admins who can manage users, roles, and authentication configuration in a product, and the admins who are allowed to provision core resources in that product (like creating and managing clusters in CockroachDB Cloud).
+
+### Is it possible to assign more than 1 role to a user in a CockroachDB Cloud organization?
+
+Yes, it is possible to assign more than one role to a user. The default minimum access role Org Member is always assigned to every user as long as they’re a part of the CockroachDB Cloud organization. Beyond that, every other assigned role is additive to the overall entitlements of a user. Best example of this is the initial user who is by default assigned the Org Member, Org Admin, Cluster Admin and Billing Coordinator [to be added soon] roles when they create the CockroachDB Cloud organization. 
+
+### Can we follow the least privilege principle by using the roles available in the CockroachDB Cloud authorization model?
+
+Yes, the roles available in the CockroachDB Cloud authorization model allow admins to grant only those entitlements to users that are supposed to map to their intended workflows. 
+
+Cluster level roles like Cluster Admin, Cluster Operator Reader & Writer [to be added soon] or Cluster Developer allow to perform pertinent actions for one or more clusters, while providing differentiation between admin, operational hybrid and non-admin entitlements. Whereas, the Organization level roles like Org Admin and Org Member allow admin and non-admin access respectively for the entire organization, while avoiding overlap with cluster specific actions.
+
+We discourage further use of the Legacy Org Developer role as that may have a broader set of entitlements than what you may want to assign to non-admins. Same would apply to Legacy Org Admin once the new Org Admin [to be added soon] role is available. The legacy roles will be deprecated after the updated CockroachDB Cloud authorization model is generally available to all customers, and will be completely removed six months after that.
```

Comment:
```suggestion
Cockroach Labs recommends that you migrate away from the legacy Org Developer and Org Admin roles to avoid over-granting privileges to non-admin members.  The legacy roles will be deprecated for six months after the updated {{ site.data.products.db }} authorization model is generally available to all customers, before being removed entirely.
```